test/util/utils.js

```diff
@@ -203,7 +219,7 @@ exports.getFilesChangedInPR = function getFilesChangedInPR() {
       console.log('>>>>> Files changed in this PR are as follows:')
       console.log(filesChanged);
       swaggerFilesInPR = filesChanged.split('\n').filter(function (item) {
-        if (item.match(/.*json$/ig) == null) {
+        if (item.match(/.*json$/ig) == null || item.match(/package.json$/ig) != null) {
```

Comment:
As in this PR I've changed `package.json` it is coming into array of changed swaggers in PR but it should not. So I added that condition but now i think i may have better condition.

Look for files those have `specification` into their path instead looking for all `.json` & then ignoring.